src/main/java/io/jenkins/plugins/opentelemetry/computer/CloudSpanNamingStrategy.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright The Original Author or Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.jenkins.plugins.opentelemetry.computer;
+
+import com.google.common.annotations.VisibleForTesting;
+import hudson.Extension;
+import hudson.slaves.NodeProvisioner;
+import jenkins.YesNoMaybe;
+import org.jenkinsci.Symbol;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Use same root span name for all pull cloud labels
+ */
+@Extension(dynamicLoadable = YesNoMaybe.YES)
+@Symbol("cloudSpanNamingStrategy")
+public class CloudSpanNamingStrategy {
```

Comment:
From:

![image](https://user-images.githubusercontent.com/2871786/124391471-adb86b80-dce8-11eb-8608-aa4d29ef07d7.png)

To:

![image](https://user-images.githubusercontent.com/2871786/124391620-5b2b7f00-dce9-11eb-8a35-1f53ab266cf4.png)